services/web/server/src/simcore_service_webserver/announcements/_models.py

```diff
@@ -18,10 +18,10 @@ class Announcement(BaseModel):
     link: str
     widgets: list[Literal["login", "ribbon", "user-menu"]]
 
-    @validator("end")
+    @field_validator("end")
     @classmethod
-    def check_start_before_end(cls, v, values):
-        if start := values.get("start"):
+    def check_start_before_end(cls, v, info: ValidationInfo):
```

Comment:
MINOR: I started protecting all access to validators since they should never be explicitly called. i.e. here I would rename as `_check_start_before_end` (i.e. with `_` prefix)